app/routes/apply/new-eligibility/date-of-birth.js

```diff
@@ -1,30 +1,50 @@
 const DateOfBirth = require('../../../services/domain/date-of-birth')
 const UrlPathValidator = require('../../../services/validators/url-path-validator')
 const ValidationError = require('../../../services/errors/validation-error')
+const ERROR_MESSAGES = require('../../../services/validators/validation-error-messages')
+const claimTypeEnum = require('../../../constants/claim-type-enum')
 
 module.exports = function (router) {
   router.get('/apply/:claimType/new-eligibility', function (req, res) {
     UrlPathValidator(req.params)
+    var errors
+
+    req.session.claimType = claimTypeEnum.FIRST_TIME
```

Comment:
this should not be hard coded, will break repeat-new-eligibility claims made thro you-claims flow